app/models/foreman_tasks/concerns/action_triggering.rb

```diff
@@ -3,12 +3,10 @@ module Concerns
     module ActionTriggering
       extend ActiveSupport::Concern
 
-      included do
```

Comment:
Since this is being included from other plugins, I think we should at least raise some expalanationary exception here, warning the users that they should use `prepend` instead of `include`